packages/node_modules/@webex/plugin-meetings/src/meetings/collection.js

```diff
@@ -43,7 +44,7 @@ const MeetingCollection = {
    * @returns {Object} returns an object map of Meeting instances
    * @memberof MeetingCollection
    */
-  getAll(options) {
+  getAll(options = false) {
```

Comment:
options should probably default to an empty object here instead of a boolean to keep it consistent.

Reply:
ya missed it can i change it to {active: false}